packages/openssl.rb

```diff
@@ -1,12 +1,64 @@
 require 'package'
 
 class Openssl < Package
-  description 'OpenSSL is a robust, commercial-grade, and full-featured toolkit for the Transport Layer Security (TLS) and Secure Sockets Layer (SSL) protocols.'
-  homepage 'https://www.openssl.org/'
-  version '3.0.2'
+  description 'The Open Source toolkit for Secure Sockets Layer and Transport Layer Security'
+  homepage 'https://www.openssl.org'
+  _ver = '1.1.1i'
+  version _ver
```

Comment:
The `_ver` variable should add the symbol `@` at first (To set the variable in this class only, not whole `package.rb`)
```ruby
@_ver = '1.1.1i'
```

Reply:
Why not whole ```package.rb```? It's conceivable that in the future ```package.rb``` might use it, no?

Reply:
If there have more than one package using the `_var` variable, it may corrupt `crew upgrade`

Reply:
Ah! I will change it.

Reply:
@uberhacker thoughts on ```s/_ver/@_ver/g``` ?

Reply:
And could ENV variable leakage be responsible for ENV variables from one package install leaking into another package install? e.g. during installs of dependencies before compiling a package with dependencies?

Reply:
The advantage of using `@` is it makes the variable scope an instance which can be used in all methods within the class.  So, for "instance", you could use the variable in the build and install functions.  More info here: https://stackoverflow.com/questions/14319347/variables-in-ruby-on-rails